src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationConstants.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace AspNet.Security.OAuth.Alipay
```

Comment:
This file, and all the other `.cs` files, need the Apache 2.0 license header adding like the rest of the files in the repo:

https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/1a32506eedbacd83fed18b985aa11d58928192d7/src/AspNet.Security.OAuth.Amazon/AmazonAuthenticationDefaults.cs#L1-L5